docs/index.md

```diff
@@ -1431,6 +1438,46 @@ we are only storing the rules whose leverage is over the 0.3 threshold.
 Both the `getItems` and the `rulesCSV` methods can also be called
 asynchronously as we saw for the `getRules` method.
 
+
+The `LocalAssociation` object can be used to retrieve the `association sets`
+related to a certain input data.
+
+
+```js
+    var bigml = require('bigml');
+    var localAssociation = new bigml.LocalAssociation(
+      'association/55922d0b37203f2a8c003010');
+    localAssociation.associationSet({product: 'cat food'},
+                                    function(error, associationSet) {
+                                      console.log(associationSet)});
+```
+
+When the
+a `LocalAssociation` instance is ready
+to predict the `LocalAssociation.associationSet`
```

Comment:
maybe better:

When the
`LocalAssociation` instance is ready the `LocalAssociation.associationSet`
method can be immediately called to predict in a synchronous way.